github/repos_traffic.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Referrer represent a referrer information.
+type Referrer struct {
+	Referrer *string `json:"referrer,omitempty"`
+	Count    *int    `json:"count,omitempty"`
+	Uniques  *int    `json:"uniques,omitempty"`
+}
+
+// Path represent a referrer information.
+type Path struct {
+	Path    *string `json:"path,omitempty"`
+	Title   *string `json:"title,omitempty"`
+	Count   *int    `json:"count,omitempty"`
+	Uniques *int    `json:"uniques,omitempty"`
+}
+
+// Time represents a time stamp as in a datapoint.
+//
+// This is necessary because github uses milliseconds timestamp.
+type Time struct {
+	time.Time
+}
+
+// UnmarshalJSON parse unix timestamp.
+func (t *Time) UnmarshalJSON(b []byte) error {
+	s := string(b)
+	i, err := strconv.ParseInt(s, 10, 64)
+	if err != nil {
+		return err
+	}
+	// We can drop the reaminder as returned values are days and it will always be 0
+	*t = Time{time.Unix(i/1000, 0)}
+	return nil
+}
+
+// Datapoint represent a view in views on clones.
+type Datapoint struct {
```

Comment:
Oh yeah sorry.

I renamed it, although it make it a bit less clearer for it's use in `TrafficClones`, not really inspired for a better name.